js/form.js

```diff
@@ -82,6 +82,15 @@ document.addEventListener("DOMContentLoaded", async () => {
               });
             }
           });
+        } else {
+          // re-enable any de-selected packages that are mandatory
+          config.packages.forEach(pkg => {
+            if (pkg.required) {
+              document.querySelector(
+                  '[data-field="packages:' + pkg.code + '"]'
+              ).checked = true;
+            }
+          })
```

Comment:
Very good and valuable feature! Maybe we can also disable the checkboxes that are required so that the user gets visual feedback. Feel free to create a separate ticket for that and assign it to me if you think that this is out of scope for this PR.

Reply:
Good idea. Done.